test/rule-matches/data-table-matches.js

```diff
@@ -0,0 +1,38 @@
+describe('data-table-matches', function () {
```

Comment:
This test should have a "should return false" case as well.

Reply:
Added, and I fixed the typos in the first two test names.